saleor/payment/gateways/np_atobarai/__init__.py

```diff
@@ -0,0 +1,70 @@
+import os
+from typing import List
+
+import opentracing
+
+from ... import TransactionKind
+from ...interface import GatewayConfig, GatewayResponse, PaymentData
+from . import api
+from .api_types import ApiConfig, PaymentStatus, get_api_config
+
+
+def inject_api_config(fun):
+    def inner(payment_information: PaymentData, config: GatewayConfig):
+        return fun(payment_information, get_api_config(config.connection_params))
+
+    return inner
+
+
+def parse_errors(errors: List[str]) -> str:
+    # FIXME: better solution?
```

Comment:
What about this? If it's not going to be fixed in this PR do we have a task for it?

Reply:
It's not going to be fixed in this PR, I've added a task in Net Protections task list.